Dockerfile

```diff
@@ -0,0 +1,44 @@
+# DESCRIPTION:	  Run Windows 95 in a container
+# AUTHOR:		  Paul DeCarlo <toolboc@gmail.com>
+#
+#   Made possible through prior art by:
+#   copy (v86 - x86 virtualization in JavaScript) 
+#   felixrieseberg (Windows95 running in electron) 
+#   Microsoft (Windows 95)
+#
+#   ***Docker Run Command***
+#
+#   docker run -it \
+#    -v /tmp/.X11-unix:/tmp/.X11-unix \ # mount the X11 socket
+#    -e DISPLAY=unix$DISPLAY \ # pass the display
+#    --device /dev/snd \ # sound
+#    --name windows95 \
+#    toolboc/windows95
+#
+#   ***TroubleShooting***
+#   If you receive Gtk-WARNING **: cannot open display: unix:0
+#   Run:
+#       xhost +
+#
+
+FROM node:10.9-stretch
+
+LABEL maintainer "Paul DeCarlo <toolboc@gmail.com>"
+
+RUN apt-get update && apt-get install -y \
```

Comment:
Is there a reason for using `apt-get` instead of the newer and better `apt`?

Reply:
*Updated to the newer better  `apt`